index.html

```diff
@@ -0,0 +1,195 @@
+          <li><button type="button">Contact Us</button></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="home-intro flex">
+        <div class="intro-heading">
+          <h2>Welcome! Food Lovers!</h2>
+          <h1>
+            Rejovan's Kitchen <br />
+            Serves Delicious <br />
+            Meal
+          </h1>
+        </div>
+        <p class="mobile-paragraph">
+          We give you the best meal you can ever think of, our customer is our
+          priority, come lets give you that special treat you crave for.
+        </p>
+        <h2 class="mobile-date">we open Mon(8:am-8:pm)- Sun(12noon-9:pm)</h2>
+        <p class="mobile-location">
+          located at arkmill road, iyana-ipaja Lagos State.
+        </p>
+      </section>
+
+      <section class="mainprogram flex">
+        <div class="header flex">
+          <h3>our feature</h3>
+          <hr />
+        </div>
+
+        <div class="activities flex">
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/drinks-1.png" alt="featureicon" />
+              <h3>Food</h3>
+            </div>
+            <p>we serve delicious meal that will satisfy your cravings</p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/chef-icon.png" alt="featureicon" />
+              <h3>Chefs</h3>
+            </div>
+            <p>
+              we have well trained chefs that are very professional both cooking
+              and serving.
+            </p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/dish-plate.png" alt="featureicon" />
+              <h3>Customer service</h3>
+            </div>
+            <p>we have a well relateable customer services.</p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/kitchenware_icon.png" alt="featureicon" />
+              <h3>Equipment</h3>
+            </div>
+            <p>
+              our cooking materials are well structured and gotten from the
+              best. And also a neat working environment.
+            </p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/cook-pot.png" alt="featureicon" />
+              <h3>Feedback</h3>
+            </div>
+            <p>we've had a lot of good reports from our costumers.</p>
+          </div>
+        </div>
+
+        <button type="button">See other feedback</button>
+        <a href="#" class="desktop-mainprogramBtn">place your order</a>
+      </section>
+    </main>
+
+    <section class="section-menu flex">
+      <div class="flex menuHeader">
+        <h3>our Menu</h3>
+        <hr />
+      </div>
```

Comment:
- [ ] Please add a background image to the foods. The background image doesn't have to be the same.
- [ ] well done with adding more buttons even if it is optional. 🥇 
- [ ] Texts are near the picture in the mobile version. Kindly change it.
- [ ] Please give more right and left space in the desktop version.
- [ ] Also pictures are close to each other in the original. Kindly fix it.
___
![image](https://user-images.githubusercontent.com/38635158/175649170-3f5e4d29-4788-4697-83d1-12c47a08d2e8.png)
___
### mobile view
![image](https://user-images.githubusercontent.com/38635158/175649265-8a62d31c-d3c8-45ac-ba91-297e72511f54.png)

___
### Original
![image](https://user-images.githubusercontent.com/38635158/175649079-c9616ce3-35d8-436a-832b-23b3524b5abb.png)
___
### Original mobile view
![image](https://user-images.githubusercontent.com/38635158/175649772-928068af-7eba-41b1-bce2-4f536c06178f.png)